doc/conda_environment.yml

```
@@ -0,0 +1,7 @@
+name: nestml-doc
+dependencies:
+  - python>=3.6
+  - pip
+  - pip:
+    - -r file:../requirements.txt
```

I think this should be
```suggestion
    - -r file:requirements.txt
```
and that file (i.e. probably a new one under `doc`) should contain the sphinx requirements. Right now, this will install the dependencies for _running_ NESTML, wouldn't it?